src/server/cloak-server.js

```diff
@@ -1,9 +1,27 @@
 var cloak = require('cloak');
 
+var listOfUsers = [];
```

Comment:
Is it possible to update  file to be es6y...so using `let`, `const` and arrow function etc. I'm pretty sure node supports it